worker-pools.yml

```diff
@@ -2958,6 +2958,40 @@ pools:
               type: SCRATCH
               interface: NVME
           machine_type: c2-standard-4
+  - pool_id: '{pool-group}/t-linux-2404-d2g'
```

Comment:
While we're standardizing on new pool names.. should we change this from `linux-2404` -> `ubuntu-2404`? It's always mildly bugged me.

Reply:
Maybe.  I don't really like having the 2404 in there to be honest, in theory it's not relevant to the consumers since everything runs in docker.  But I was struggling to come up with a name for `t-linux-{,x}large-gcp`'s successor...

Reply:
Makes sense, what's stopping us from dropping the `2404` exactly? Doesn't the `d2g` already differentiate it enough?